### PR TITLE
technical advisory 48860

### PR DESCRIPTION
**Pull request by taroface**

### CODE REVIEW

File: _data/advisories.yml [hunk 1]
```diff
@@ -1,3 +1,7 @@
+- advisory: 48860
+  summary: Data corruption/loss issue with snapshots and delete range
+  versions: 2.1.0-2.1.9, 19.1.0-19.1.8, 19.2.0-19.2.8
```
Comment (dbist) [inline _data/advisories.yml:3]: 19.2.0-19.2.6

File: advisories/a48860.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+A defect in RocksDB allowed it to violate its guarantees for snapshots, causing corruption under specific and extremely rare circumstances. This corruption is irrecoverable using regular node failover and must be addressed by restoring a backup when it occurs.
+
+This bug has been fixed in versions v19.1.9, v19.2.7, and v20.1.0. All users are encouraged to [upgrade](../stable/upgrade-cockroach-version.html).
+
+The public issue is tracked as [#48860](https://github.com/cockroachdb/cockroach/issues/48860).
+
+## Mitigation
+
+There is no mitigation possible without an upgrade. Cockroach Labs recommends that customers upgrade to the maintenance release CockroachDB v19.1.9 or v19.2.7, or upgrade to v20.1.
```
Comment (dbist) [inline advisories/a48860.md:23]: upgrade to v20.1.0.

File: _data/advisories.yml (hunk 1 above)
Comment (jseldess) [reply _data/advisories.yml]: Done.

File: advisories/a48860.md (hunk 2 above)
Comment (jseldess) [reply advisories/a48860.md]: Done.

File: advisories/a48860.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+## Description
+
+Range deletions (as caused by certain [table drops](../v20.1/drop-table.html), [table truncates](../v20.1/truncate.html), and certain rebalancing operations during [node decommissioning](../v20.1/remove-nodes.html) or failover), when occurring concurrently with replica [garbage collection](../v20.1/architecture/storage-layer.html#garbage-collection), can yield permanent data corruption or loss in rare cases.
+
+## Statement
+
+A defect in RocksDB allowed it to violate its guarantees for snapshots, causing corruption under specific and extremely rare circumstances. This corruption is irrecoverable using regular node failover and must be addressed by restoring a backup when it occurs.
+
+This bug has been fixed in versions v19.1.9, v19.2.7, and v20.1.0. All users are encouraged to [upgrade](../v20.1/upgrade-cockroach-version.html).
```
Comment (dbist) [inline advisories/a48860.md:17]: should we encourage upgrades based on the target release? Otherwise we are promoting v20.1 upgrade docs.

Comment (jseldess) [reply advisories/a48860.md]: Fixed.

**Pull request merged**